index.html

```diff
@@ -15,15 +15,22 @@
 <main class="main-container">
     <div class="card">
         <div class="card-image">
-
+            <figure class="Joke">
+                <img class="Petrosyan" src="images/401x512_0xd42ee430_2359835871452856838.png" alt="Я и без того знаю, какой ты Петросян">
+                <figcaption>
+                    Петросян
+                </figcaption>
+                </figure>
         </div>
         <div class="card-text">
             <p class="card-text-p">
-                Здесь будет текст карточки!
+                В новом 2023 году ты очевидно думал, что холодно будет в Европе
             </p>
         </div>
         <div class="card-audio">
+            <audio class="Turning-point" src="audio/Directed_by_Robert_B._Weide_-_Theme_meme.mp3">
```

Comment:
Class must match parent node. For example `card-audio-element`.
Don't forget to rename file and move both of folders `images` and `audio` inside `media` folder